docs/exercises/5-multi_region.rst

```diff
@@ -348,14 +354,15 @@ in us-west-2.
 
     .. group-tab:: Java
 
+
         We have built a simple bash script that sets the grant, thereby disabling the use of the CMK in us-east-2.
```

Comment:
should be CMK in us-west-2

Reply:
Per offline discussion, fixing this in the code update pass